app3/Main.hs

```diff
@@ -69,10 +69,10 @@ runExecuteIO (Free step) = do
           readFile (getTableFilePath tableName) >>= return . next
         runStep (Lib3.SaveFile tableName fileContent next) = 
           writeFile (getTableFilePath tableName) fileContent >>= return . next
-        runStep (Lib3.ExecutePure sql next) = do
-          currentTime <- getCurrentTime
-          let result = Lib3.parseStatement sql >>= (\parsedStmt -> Lib3.executeStatement currentTime parsedStmt)
-          return $ next result
+        runStep (Lib3.LoadAndParseTable tableName next) = do
```

Comment:
I think if this is here, then we do not need the load file function

Reply:
but then we should also probably change SaveFile to just pass DataFrame instead of FileContent